src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -43,15 +43,27 @@ public static void runJavaClass(String javaPath) {
     public static void runJavaClass(Class classToRun) {
         String className = classToRun.getName();
         className = className.replace('.', '/');
-        className += ".java";
         runJavaClass(className);
     }
 
     /**
      * @param pythonPath The path from the java package to the class.
-     *                   example: "keyboard/keyboard_to_nt.py"
+     *                   example: "keyboard_to_nt"
```

Comment:
same about all examples, do use real files but example files like my_python_file , MyJavaFile and stuff like this, because when you do like it's now every time u will change the file you will need to change the example